typescript-test-samples/schema-and-contract-testing/README.md

```diff
@@ -0,0 +1,109 @@
+[![typescript: 4.5.5](https://badgen.net/badge/Built%20With/TypeScript/blue9)](https://badgen.net/badge/Built%20With/TypeScript/blue9)
+[![test: unit](https://img.shields.io/badge/Test-Unit-blue)](https://img.shields.io/badge/Test-Unit-blue)
+
+# Typescript: Schema and Contract testing for Event-Driven Architectures
+
+---
+
+## Contents
+
+- [Introduction](#introduction)
+- [Schema testing](#schema-testing-schema-difference)
+  - [Goal](#goal)
+  - [Description](#description)
+  - [System under test](#system-under-test-sut)
+  - [Limitations](#limitations)
+- [Contract testing](#contract-testing)
+  - [Goal](#goal-1)
+  - [Description](#description-1)
+  - [System under test](#system-under-test-sut-1)
+  - [Limitations](#limitations-1)
+
+
+- [Key Files in the Project](#key-files-in-the-project)
```

Comment:
Link is broken

Reply:
fixed. I swear I tested all the links, maybe it was a last-minute change :)